docs/openapi_generator/pyopenapi/operations.py

```diff
@@ -114,10 +114,14 @@ class EndpointOperation:
     response_examples: Optional[List[Any]] = None
 
     def get_route(self) -> str:
+        # Get the API level from the webmethod decorator
+        webmethod = getattr(self.func_ref, "__webmethod__", None)
+        api_level = webmethod.level if webmethod and hasattr(webmethod, 'level') else LLAMA_STACK_API_V1
```

Comment:
Hmmmm, do we want to assume `v1` if no level is specified? Perhaps `v1alpha` would be a better option?